CHANGES/6530.bugfix

```diff
@@ -0,0 +1 @@
+Do not publish repodata that has artifacts in the root directory.
```

Comment:
Fixed publishing of module repodata ?
Publish metadata in the repodata directory only ?
Stopped publishing modular repodata in the root directory of a publication ?